src/mpi/romio/adio/ad_gpfs/ad_gpfs_rdcoll.c

```diff
@@ -1163,7 +1163,7 @@ static void ADIOI_R_Exchange_data_alltoallv(ADIO_File fd, void *buf, ADIOI_Flatl
         if (rdispls[i - 1] != rdispls[i]) {
             DBG_FPRINTF(stderr, "\t\t[%d]%2ld,", i, rdispls[i]);
         }
-    DBG_FPRINTF(stderr, "\ttails = %4d, %4d\n", stail, rtail);
+    DBG_FPRINTF(stderr, "\ttails = %4lld, %4lld\n", stail, rtail);
```

Comment:
```suggestion
    DBG_FPRINTF(stderr, "\ttails = %4lld, %4lld\n", (long long) stail, (long long) rtail);
```
MPI_Count may be `long`.

Reply:
Thanks for the reminder. Fixed!